compiler/rustc_hir/src/lang_items.rs

```diff
@@ -38,27 +38,34 @@ macro_rules! expand_group {
 // So you probably just want to nip down to the end.
 macro_rules! language_item_table {
     (
-        $( $variant:ident $($group:expr)?, $name:expr, $method:ident, $target:expr; )*
+        $( $(#[attr:meta])* $variant:ident $($group:expr)?, $name:expr, $method:ident, $target:expr; )*
```

Comment:
> Hmm, not quite right:

You can separate the path from the item:
```suggestion
        $( $(#[$attr:meta])* $variant:ident $($group:expr)?, $module:ident :: $name:ident, $method:ident, $target:expr; )*
```
and then use it like `$module::$name` below where it uses `$name` now, except for the `stringify!` call.

Reply:
Ah, good idea! Hopefully this doesn't have the potential to confuse people already familiar with how this works; I think it should be okay because they should get name resolution errors if they use it incorrectly.